lib/modules/manager/npm/extract/index.ts

```diff
@@ -338,6 +339,46 @@ export async function extractPackageFile(
     return dep;
   }
 
+  /**
+   * Used when there is a json object as a value in overrides block.
+   * @param parents
+   * @param child
+   * @returns PackageDependency array
+   */
+  function extractOverrideDepsRec(
+    parents: string[],
+    child: NpmPackageDependency
+  ): PackageDependency[] {
+    const deps: PackageDependency[] = [];
+    if (!child || is.emptyObject(child)) {
+      return deps;
+    }
+    for (const [key, val] of Object.entries(child)) {
+      if (is.string(val)) {
+        const dep: PackageDependency = {};
+        // special handling for "." key
+        // "." means the constraint is applied to the parent dep
+        dep.depName = key === '.' ? parents[parents.length - 1] : key;
+        dep.groupName = 'overrides';
```

Comment:
are we sure we always like to group it as `overrides`?

@rarkins WDYT?

Reply:
Removed it for it.